README.md

```diff
@@ -18,7 +18,6 @@ Requirements
 
 - A working python installation (tested to work with python 2.7).
```

Comment:
this mention python 2.7, could be update to 3.11

Reply:
sure, updated.